docs/user-documentation/accessibility.md

```diff
@@ -0,0 +1,3 @@
+Islandora 8 leans heavily on Drupal for accessibility. Drupal 8 core comes with a number of features to support greater accessibility for developers, site builders, and end users, as outlined [here](https://www.drupal.org/docs/8/accessibility/drupal-8-accessibility-features). This can be extended further with contributed modules, as outlined [here](https://www.drupal.org/docs/8/accessibility/contributed-modules-for-extending-accessibility-in-drupal-8) 
```

Comment:
I like that we are putting this in here and don't want to block it. However, I think rephrasing to avoid using the word "here" as link text may be advisable by said accessibility guidelines. For example:


> Islandora 8's accessibility features are provided by Drupal 8 and contributed Drupal 8 modules. These pages within the Accessibility section of the Drupal documentation can provide guidance:
> 
> - [Drupal 8 Accessibility Features](https://www.drupal.org/docs/8/accessibility/drupal-8-accessibility-features)
> - [Contributed Modules for Extending Accessibility in Drupal 8](https://www.drupal.org/docs/8/accessibility/contributed-modules-for-extending-accessibility-in-drupal-8)
> - [Hide Content Properly (for all users including users with screen readers)](https://www.drupal.org/docs/8/accessibility/hide-content-properly)
> - [How to do an accessibility review?](https://www.drupal.org/docs/8/accessibility/how-to-do-an-accessibility-review)
> - List of [External Accessibility Resources](https://www.drupal.org/docs/8/accessibility/external-accessibility-resources)